en/identity-server/next/mkdocs.yml

```diff
@@ -674,7 +674,7 @@ nav:
         - Build self-service capabilities: guides/user-self-service/build-self-service-capabilities.md
       - Organizations:
           - Organizations: guides/organization-management/index.md
-          - Try a B2B use case: guides/organization-management/try-a-b2b-use-case.md
+          - Try a B2B use case: guides/organization-management/try-a-b2b-use-case-pet-care.md
```

Comment:
Don't we need to update the file path of [/en/identity-server/next/docs/guides/organization-management/try-a-b2b-use-case.md](https://github.com/amanda-ariyaratne/docs-is/blob/pet-care/en/identity-server/next/docs/guides/organization-management/try-a-b2b-use-case.md) with this change?

Reply:
Resolved with https://github.com/wso2/docs-is/pull/5204/commits/c1ac4d5456f0a8a63d1fd8b8b20ec40da3543afb